tests/FakeItEasy.Specs/WrappingFakeSpecs.cs

```diff
@@ -272,6 +274,56 @@ public static void NotAWrappingFakeCallsWrappedMethod(IFoo fake, Exception excep
                     .WithMessage("The configured fake is not a wrapping fake."));
         }
 
+        [Scenario]
+        public static void EventSubscriptionSubscribesToWrappedEvent(Foo realObject, IFoo wrapper, EventHandler handler)
+        {
+            "Given a real object"
+                .x(() => realObject = new Foo());
+
+            "And a fake wrapping this object"
+                .x(() => wrapper = A.Fake<IFoo>(o => o.Wrapping(realObject)));
+
+            "And an event handler"
+                .x(() => handler = A.Fake<EventHandler>());
+
+            "When the handler is subscribed to the fake's event"
+                .x(() => wrapper.SomeEvent += handler);
+
+            "And the event is raised using a method on the wrapped object"
+                .x(() => realObject.RaiseSomeEvent());
+
+            "Then the handler is called"
+                .x(() => A.CallTo(handler).MustHaveHappened());
+        }
+
+        [Scenario]
+        public static void RaisingEventOnWrappingFakeUsingRaiseSyntax(Foo realObject, IFoo wrapper, EventHandler handler, Exception exception)
+        {
+            "Given a real object"
+                .x(() => realObject = new Foo());
+
+            "And a fake wrapping this object"
+                .x(() => wrapper = A.Fake<IFoo>(o => o.Wrapping(realObject)));
+
+            "And an event handler"
+                .x(() => handler = A.Fake<EventHandler>());
+
+            "When the handler is subscribed to the fake's event"
+                .x(() => wrapper.SomeEvent += handler);
+
+            "And the event is raised using the '+= Raise' syntax"
+                .x(() => realObject.SomeEvent += Raise.WithEmpty());
+
+            "Then the handler is not called"
+                .x(() => A.CallTo(handler).MustNotHaveHappened());
+
+            "And when the event is raised using a method on the wrapped object"
+                .x(() => exception = Record.Exception(() => realObject.RaiseSomeEvent()));
+
+            "Then a NotSupportedException is thrown"
+                .x(() => exception.Should().BeAnExceptionOfType<NotSupportedException>());
+        }
+

```

Comment:
Could we also add the sample code you had above (

```c#
EventHandler handler = null;
A.CallTo(fake).Where(call => call.Method.Name == "add_Event").Invokes((EventHandler h) => handler += h);
A.CallTo(fake).Where(call => call.Method.Name == "remove_Event").Invokes((EventHandler h) => handler -= h)
```
) or similar to either a test or an example so we can point to it in case people want to raise events from a wrapped Fake? I guess it's pretty unlikely, but still.

Reply:
Will do. We could also add it to the docs, I guess.